mobile/library/cc/engine_builder.cc

```diff
@@ -242,6 +244,23 @@ EngineBuilder& EngineBuilder::enforceTrustChainVerification(bool trust_chain_ver
   return *this;
 }
 
+EngineBuilder& EngineBuilder::addRtdsLayer(const std::string& layer_name, int timeout_seconds) {
+  rtds_layer_name_ = layer_name;
+  rtds_timeout_seconds_ = timeout_seconds;
+  return *this;
+}
+EngineBuilder& EngineBuilder::setAggregatedDiscoveryService(const std::string& api_type,
+                                                            const std::string& address,
+                                                            const int port) {
+#ifndef ENVOY_GOOGLE_GRPC
+  raise std::runtime_error("google_grpc must be enabled in bazel to use ADS");
```

Comment:
what does `raise` do in C++?  shouldn't this be `throw`?  not familiar with `raise` in c++.

Reply:
Ah good catch